centinel/data/country.txt

```diff
@@ -0,0 +1,2 @@
+"yahoo.fr/this,and,that",xx,yy
```

Comment:
I don't think commas are legal in URLs-> you use & to distinguish between url parameters, which are key value pairs. What is the purpose of this line?